src/components/activities/newPatientActivity/NewPatientActivity.tsx

```diff
@@ -108,42 +108,44 @@ const NewPatientActivity: FunctionComponent<TProps> = ({
           <div className="newPatient__background">
             <div className="newPatient__content">
               <div className="newPatient__title">{t("nav.newpatient")}</div>
-              <PatientDataForm
-                fields={initialFields}
-                onSubmit={onSubmit}
-                submitButtonLabel={t("common.submit")}
-                resetButtonLabel={t("common.clearall")}
-                isLoading={isLoading}
-                shouldResetForm={shouldResetForm}
-                resetFormCallback={resetFormCallback}
+              <PermissionWrapper permissions={["patient.create"]}>
+                <PatientDataForm
+                  fields={initialFields}
+                  onSubmit={onSubmit}
+                  submitButtonLabel={t("common.submit")}
+                  resetButtonLabel={t("common.clearall")}
+                  isLoading={isLoading}
+                  shouldResetForm={shouldResetForm}
+                  resetFormCallback={resetFormCallback}
+                />
+              </PermissionWrapper>
+              <div ref={infoBoxRef}>
```

Comment:
I think that the wrapper's position is in the wrong place. I mean the patient create permission should check in three different points:
- new patient button inside the patient dashboard.
- new patient route (some control that prevent the user to access to that route)
- a check inside the form page (it's useless in due to the two previous check but I think would be great to have this too). 

The third check should wrap the entire page's UI and not only the form it's self.